python/python/glide/async_commands/core.py

```diff
@@ -2326,6 +2326,33 @@ async def zscore(self, key: str, member: str) -> Optional[float]:
             await self._execute_command(RequestType.ZScore, [key, member]),
         )
 
+    async def zmscore(
+        self,
+        key: str,
+        members: List[str],
+    ) -> List[float]:
+        """
+        Returns the scores associated with the specified `members` in the sorted set stored at `key`.
+
+        See https://redis.io/commands/zmscore for more details.
+
+        Args:
+            key (str): The key of the sorted set.
+            members (List[str]): A list of members in the sorted set.
+
+        Returns:
+            List[float]: A list of scores of the `members`.
+                If a `member` does not exist, the corresponding value in the list will be `None`.
+
+        Examples:
+            >>> await client.zmscore("my_sorted_set", ["one", "non_existent_member", "three"])
+                [1.0, None, 3.0]
+        """
+        return cast(
+            List[float],
```

Comment:
Maybe
```suggestion
            List[Optional[float]],
```

if yes - update doc and return type and transaction

Reply:
The redis docs say "null reply: if the member does not exist in the sorted set", however it does not seem like a null value is ever returned when I use redis-cli. If I specify a non-existing key I get `1) (nil)` as a response instead of just `(nil)`, and if I specify an existing key with one non-existing member I also get `1) (nil)`

Reply:
oh right, thank you for clarifying things!